Update managing-environments.md (#1053)
fishtown-analytics/docs.getdbt.com: website/docs/docs/guides/managing-environments.md

Targets offer the flexibility to decide how to implement your separate environments – whether you want to use separate schemas, databases, or entirely different clusters altogether! We recommend using _different schemas within one data warehouse_ to separate your environments. This is the easiest to set up, and is the most cost effective solution in a modern cloud-based data stack.

In practice, this means that most of the details in a target will be consistent across all targets, except for the `schema` and user credentials. If you have multiple dbt users writing code, it often makes sense for _each user_ to have their own _development_ environment. A pattern we've found useful is to set your dev target schema to be `dbt_<username>`. User credentials should also differ across targets so that each dbt user is using their own data warehouse user.

> :::info Deleting Environments

Reviewer: This is a great addition, @mirnawong1! What do you think about making it its own section? I feel it would be easier to see because it will be n the TOC, and easier to find when you scan the page.
```suggestion
## Deleting environments
```

> A quick note on deleting Environments. Deleting an Environment will not automically delete the job(s) associated to it. Before you completely delete your Environment, remember to delete any jobs associated to that Environment first. Once you've deleted the job, you can delete the Environment fully. 

Reviewer: We can just lowercase "environment" because it's not the name of a feature. I also made some rewording suggestions.
```suggestion
Deleting an environment will not automatically delete the associated job(s). Before you completely delete your environment, first delete any jobs associated to that environment. Once you've deleted the job(s), you can completely delete the environment. 
```

> :::

Reviewer: ```suggestion
```

Author: hi @runleonarun and @amychen1776 ! thank you both so much and this all makes sense to me! 

just to address @amychen1776's question -  do you both think the entire environments page should also live in the [dbt cloud page](https://docs.getdbt.com/docs/dbt-cloud/using-dbt-cloud/cloud-notifications) as well? or is this duplicative

Reviewer 2: I personally think it's time to start a new page in dbt cloud about environments because the type of environments we are referring on this page is not exactly the dbt Cloud feature "environments" (even though they are directly related).  I also have a writeup that updates this page that we are still trying to figure out on how to implement.

Author: ah yes I see - and it definitely makes sense to have a separate dbt cloud environment page. could it live [here](https://docs.getdbt.com/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-choosing-a-dbt-version) in the interim @amychen1776 and @runleonarun and on the 'managing environment' page?

Reviewer 2: I'll leave that decision up to Leona :)
